apps/docs/components/SideNavigation.tsx

```diff
@@ -0,0 +1,80 @@
+import useTableContentObserver from "@/hooks/useTableContentObserver";
+import Link from "next/link";
+import { useEffect, useState } from "react";
+
+type Heading = {
+  id: string;
+  text: string | null;
+  level: number;
+};
+
+const SideNavigation = ({ pathname }) => {
```

Comment:
please use named export for consistency in the codebase 🙏

Reply:
Done